projects/js-packages/publicize-components/src/components/social-image-generator/panel/styles.module.scss

```
@@ -8,4 +11,20 @@
 
 .mediaPicker {
 	max-width: 150px;
-}
\ No newline at end of file
+}
+
+.footer {
+	display: flex;
+	justify-content: flex-end;
+
+	> * + * {
+		margin-left: var( --spacing-base );
```

Let us make it [RTL aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/) by making use of [Logical Properties](https://www.w3.org/TR/css-logical-1/).

```suggestion
		margin-inline-start: var( --spacing-base );
```

---

Thanks, fixed in fec966c